lib/formats/json/json.flow

```diff
@@ -96,8 +96,7 @@ export {
 }
 
 parseJson(json : string) -> Json {
-	v = doParseJson(json, strlen(json), 0);
-	v.first;
+	parseJsonSafe(json)
```

Comment:
It seems we are just switching to the other one? Is that really the point?

Reply:
No, parseJsonSafe contains exactly those lines I removed, so nothing changes in means of functioning. In JS, this function is executed natively and the body defined is ignored anyway.

Reply:
No, parseJsonSafe contains exactly those lines I removed, so nothing changes in means of functioning. In JS, this function is executed natively and the body defined is ignored anyway.